### DISCUSSION TITLE
Help with NerdFont

### DISCUSSION THREAD

**Discussion by aakashrshah** [starship/starship#5182] [Q&A]
on  master [$✘!?] is 📦 v0.1.0 via 🐍 v3.10.10 

I am unable to get the trunk symbol. Even after i set it on my Terminal from NerdFontSymbolOnly package. 

This is really annoying me for some reason (idk why)

https://stackoverflow.com/questions/64072632/getting-a-and-a-instead-of-just-the-branch-icon-and-the-branch-name-on

I tried setting just one for master branch in.

I would also like to have the default symbols available in the ~/.config/starship.toml, no? so that i can reset to original state?

**Comment by aakashrshah** [comment]
i see in the docs - 

```
while the second should produce a [powerline branch symbol (e0a0) (opens new window)](https://github.com/ryanoasis/powerline-extra-symbols#glyphs).
```

I get the snake symbol but not the powerline-extra symbol

**Comment by mickimnet** [comment]
What terminal are you using, and what font are you using?

**Reply by aakashrshah** [reply]
Mac OS,  VSCode terminal, zsh. I am using the FiraCode. But i think i can set either the Symbol or the Font in MacOS Terminal, i don't know how to set both.

**Reply by mickimnet** [reply]
There is one font selection in _Apple Terminal_, where you should select _FiraCode_. Besides, I wouldn't recommend using _Apple Terminal_, as it doesn't support true color (24bit). I'm using _iTermn_ on _macOS_.

Regarding, _Visual Studio Code_ you need to set _FiraCode_ either for the editor or for the terminal as well. Search for "terminal font" in the settings of _Visual Studio Code_.

**Reply by aakashrshah** [reply]
So do you mean FiraCode has the symbols as well?

I am using Firacode in VSCode and terminal. But I am still not seeing symbol for master

<img width="472" alt="Screenshot 2023-05-16 at 3 00 16 PM" src="https://github.com/starship/starship/assets/3517550/fa684e4f-65ed-4399-a9b5-edff452a8eeb">

**Reply by mickimnet** [reply]
Oh! No, you need a special _Nerd Fonts_ version of _FiraCode_ or any other font for that matter. You can get them from the [Download page](https://www.nerdfonts.com/font-downloads) on the _Nerd Fonts_ page.

**Reply by aakashrshah** [reply]
I appreciate your help! I really do.

I don’t know which one to download. They are either fonts or symbols. I don’t know which exactly are you referring to

**Reply by mickimnet** [reply]
All of them are fonts **with** symbols. I would recommend using either (the following links are direct download links):
- [FireCode Nerd Font](https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.1/FiraCode.zip)
- [Hack Nerd Font](https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.1/Hack.zip)

Extract them, install the font(s) and select one of them in your terminal or _Visual Studio Code_.
